### PR TITLE
[IOTDB-3966] [IOTDB-4079]Show leadership when show Regions && fix region id duplicated

### PR DESCRIPTION
**Pull request by RYH61**
add leadership
图片

fix region id duplicated,like nodeid duplicated[https://github.com/apache/iotdb/pull/6909](url)

### CODE REVIEW

File: node-commons/src/main/java/org/apache/iotdb/commons/cluster/RegionRoleType.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+ * under the License.
+ */
+package org.apache.iotdb.commons.cluster;
+
+/** Region Role for showing regions */
+public enum RegionRoleType {
+  // Node registered
+  Leader("Leader"),
+  // Node online ,right now Online is Running
+  Follower("Follower");
```
Comment (CRZbulabula) [inline node-commons/src/main/java/org/apache/iotdb/commons/cluster/RegionRoleType.java:26]: These two annotations seem stranged

File: confignode/src/main/java/org/apache/iotdb/confignode/manager/ConfigManager.java [hunk 2]
```diff
@@ -839,10 +841,27 @@ public UDFManager getUDFManager() {
...
     TSStatus status = confirmLeader();
     if (status.getCode() == TSStatusCode.SUCCESS_STATUS.getStatusCode()) {
-      return partitionManager.getRegionInfoList(getRegionInfoListPlan);
+      RegionInfoListResp regionInfoListResp =
+          (RegionInfoListResp) partitionManager.getRegionInfoList(getRegionInfoListPlan);
+      regionInfoListResp
+          .getRegionInfoList()
+          .forEach(
+              regionInfo -> {
+                Map<TConsensusGroupId, Integer> allLeadership = loadManager.getAllLeadership();
```
Comment (CRZbulabula) [inline confignode/src/main/java/org/apache/iotdb/confignode/manager/ConfigManager.java:853]: Please update the getAllLeadership() interface to adapt MultiLeader consensus protocol~

**Review by CRZbulabula — CHANGES_REQUESTED** [review]
PTAL~

File: confignode/src/main/java/org/apache/iotdb/confignode/manager/ConfigManager.java (hunk 2 above)
Comment (RYH61) [reply confignode/src/main/java/org/apache/iotdb/confignode/manager/ConfigManager.java:853]: fixed~

File: node-commons/src/main/java/org/apache/iotdb/commons/cluster/RegionRoleType.java (hunk 1 above)
Comment (RYH61) [reply node-commons/src/main/java/org/apache/iotdb/commons/cluster/RegionRoleType.java]: fixed~

**Review by CRZbulabula — APPROVED** [review]
LGTM~

File: confignode/src/main/java/org/apache/iotdb/confignode/persistence/partition/PartitionInfo.java [hunk 3]
```diff
@@ -149,7 +149,7 @@ public void addMetrics() {
   }
 
   public int generateNextRegionGroupId() {
```
Comment (wangchao316) [inline confignode/src/main/java/org/apache/iotdb/confignode/persistence/partition/PartitionInfo.java:151]: "nextRegionGroupId.set(-1 )"  in  "clear()" method.

**Review by wangchao316 — CHANGES_REQUESTED** [review]

File: confignode/src/main/java/org/apache/iotdb/confignode/persistence/partition/PartitionInfo.java (hunk 3 above)
Comment (RYH61) [reply confignode/src/main/java/org/apache/iotdb/confignode/persistence/partition/PartitionInfo.java:151]: fixed~

**Review by wangchao316 — APPROVED** [review]
LGTM

**Pull request merged**